deno/rest/v10/channel.ts

```diff
@@ -366,6 +366,10 @@ export type RESTDeleteAPIChannelMessageUserReactionResult = never;
  * https://discord.com/developers/docs/resources/channel#get-reactions
  */
 export interface RESTGetAPIChannelMessageReactionUsersQuery {
+	/**
+	 * The reaction type (0 for normal and 1 for super)
```

Comment:
I would recommend removing the information in the brackets—it's already in `ReactionType`. It's not really good for future-proofing if and when more types are added... it'll just be a longer comment which is still explained by the type.